modules/cnf-provisioning-deploying-a-distributed-unit-(du)-manually.adoc

```diff
@@ -0,0 +1,130 @@
+
+== The manifest structure
+
+The profile is built from one cluster specific folder and one or more site-specific folders.
+This is done to address a deployment that includes remote worker nodes (several sites belonging to the same cluster).
+
+The [`cluster-config`](ran-profile/cluster-config) directory contains performance and PTP customizations based upon
+operator deployments in [`deploy`](../feature-configs/deploy) folder.
+
+The [`site.1.fqdn`](site.1.fqdn) folder contains site-specific network customizations.
+
+== Prerequisites
+
+Before installing the operators and deploying the DU, perform the following steps.
+
+. Clone the files in the `ran-profile` folder here:
+link:https://github.com/openshift-kni/cnf-features-deploy/tree/master/feature-configs/cn-ran-overlays/ran-profile[ran-profile].
+
+. Create a machine config pool for the RAN worker nodes. For example:
++
+[source,yaml]
+----
+cat <<EOF | oc apply -f -
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-cnf
+  labels:
+    machineconfiguration.openshift.io/role: worker-cnf
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {
+          key: machineconfiguration.openshift.io/role,
+          operator: In,
+          values: [worker-cnf, worker],
+        }
+  paused: false
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker-cnf: ""
+----
+
+. Include the worker node in the above machine config pool by labelling it with the `node-role.kubernetes.io/worker-cnf` label:
++
+----
+oc label --overwrite node/<your node name> node-role.kubernetes.io/worker-cnf=""
+----
+
+. Label the node as ptp slave (DU only):
++
+----
+oc label --overwrite node/<your node name> ptp/slave=""
+----
+
+An example of labeling the nodes used in CI/CD can be seen here:
+link:https://raw.githubusercontent.com/openshift-kni/cnf-features-deploy/master/hack/setup-test-cluster.sh[setup-test-cluster.sh].
+
+
+== Deployment
+
+The profile is built in layers with `kustomize`. To get the profile output, run:
+
+----
+oc kustomize ran-profile
+----
+
+It can be applied manually or with the toolset of your choice (for example, ArgoCD).
+
+This project contains makefile based tooling, that can be used as follows (from the project root):
+
+----
+FEATURES_ENVIRONMENT=cn-ran-overlays FEATURES=ran-profile make feature-deploy
+----
+
+== SR-IOV configuration notes
+
+The SriovNetworkNodePolicy object must be configured differently for different NIC models and placements.
+
+|====================
+|*Manufacturer* |*deviceType* |*isRdma*
+|Intel        |vfio-pci or netdevice |false
+|Mellanox     |netdevice |structure
+|====================
+
+In addition, when configuring the `nicSelector`, the `pfNames` value must match the intended interface name on the specific host.
+
+If there is a mixed cluster where some of the nodes are deployed with Intel NICs and some with Mellanox, several
```

Comment:
looks like this sentence got cut in half after "several"

Reply:
Fixed.